Doppler.Sap/appsettings.json

```
@@ -18,6 +18,7 @@
     "Password": "[SECRET_KEY]",
     "BaseServerUrl": "https://172.25.16.86:50000/b1s/v1/",
     "SlackAlertUrl": "[SECRET_KEY]",
-    "MaxAmountAllowedAccounts": 10
+    "MaxAmountAllowedAccounts": 10,
+    "SessionTimeout": 30
```

Creo que está bueno que esté explícita la unidad utilizada:

```suggestion
    "SessionTimeoutMinutes": 30
```